loqusdb/commands/load_profile.py

```diff
@@ -47,6 +54,15 @@ def load_profile(ctx, variant_file, update, stats, profile_threshold):
 
     adapter = ctx.obj['adapter']
 
```

Comment:
Could you add some log messages like

`"Running loqusdb load_profiles"`

`"Check if case is duplicated in database"`

etc